### ISSUE TITLE
sbt 1.3 regression: can't set `scalaVersion` when using `ScriptMain`

### ISSUE THREAD

**Issue by SethTisue** [sbt/sbt#4973]
## steps

here is my script:

```
#!/usr/bin/env sbt -Dsbt.version=1.2.8 -Dsbt.main.class=sbt.ScriptMain -Dsbt.supershell=false

/***
scalaVersion := "2.13.0"
*/

println(util.Properties.versionString)
```

this prints `version 2.13.0`, as expected

but if I change `1.2.8` to `1.3.0-RC4`...

## problem

...then it prints `version 2.12.8` ?!

**Comment by SethTisue** [comment]
note that it's not that the `/***` stuff is being ignored entirely. I discovered the problem in a real script with more settings. all the settings in the script took effect except for the `scalaVersion` one

**Comment by eatkins** [comment]
Thanks for the report. Should be fixed by #4988.

**Comment by eed3si9n** [comment]
Merged #4988
